src/sqlfluff/core/templaters/jinja.py

```diff
@@ -200,10 +200,10 @@ def _crawl_tree(cls, tree, variable_names, raw):
                 line_pos=pos,
             )
 
-    @staticmethod
```

Comment:
Lots of small changes in this file. The details are not interesting; basically refactoring so we always work with a Jinja environment that is aware of the `load_macros_from_path` config setting.